### ISSUE TITLE
How to classify images using convolutional neural network(CNN) in Matlab

### ISSUE THREAD

**Issue by collince777** [section-engineering-education/engineering-education#4256]
####[Machine Learning]How to classify images using convolutional neural network(CNN) in Matlab

#### Introduction
Convolutional Neural Network(CNN) is a powerful machine learning technique from the deep learning techniques from the deep learning domain. A collection of diverge image is required to train CNNs. The larger the collection, the richer the features that CNN learns.  Image classification is the process of categorizing and labeling groups of pixels or vectors within an image based on specific rules.
Training CNN with large collection of diverse image is not an easy task. However, there is an easy way. We can use pre-trained CNN to leverage the power of CNN. One advantage of image classification is that after you have learn this, you can do anything you want with it for example, you can train classifiers to identify brain tumor, cancer cell and skin diseases. In this tutorial, we will see on how to classify images using CNN in Matlab

#### Key takeaways 
- Plotting the structure of CNN.
- Finding the number of classes of a CNN.
- How to visualize the features of different layers of CNN.
- Comparing features between training data set and testing data set.
- Classifying images using CNN
#### Article quality
How is your article unique? Tell us what makes your approach different from similar articles that have published on the same topic?
Is yours more in-depth? Does it cover additional topics? Do you provide handy tips or anecdotal advice?

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by lalith1403** [comment]
Good afternoon @collince777  and thank you for submitting your topic to the EngEd program.
After some careful consideration, it struck us that this topic may be a bit over-saturated throughout other blog sites and official documentations.

We typically refrain from publishing content that is covered widely on the net or other blogs. We're more interested in original, practitioner-focused content that takes a deeper dive into programming-centric concepts.

But in order to approve the topic, it has to serve value to the larger developer community at large.
An option and a great way to write this as an in-depth article and make it more add value to the greater developer community at large would be to walk the reader through the USE of methods and functions by building a unique, different, useful project.

> We would love to learn your thought process behind the solution you arrived at using these concepts and topics.

That way a developer could see them in action.
As mentioned above - we believe this topic is widely covered on other blog sites.

The best way for students to build a great portfolio is by building what does not exist and what can provide the most value.
